contracts/standard/arbitration/AppealableArbitrator.sol

```diff
@@ -1 +1,58 @@
-// TODO: Create a class allowing appeal to another arbitrator.
+/**
+ *  @title Arbitration Standard 
+ *  @author Mahadevan K - <mahadevan.k@gmail.com>
+ *  Bug Bounties: This code hasn't undertaken a bug bounty program yet.
+ */
+
+pragma solidity ^0.4.15;
+
+import "./CentralizedArbitrator.sol";
+
+/** @title Appealable Arbitrator
+ *  This is a single arbitrator deciding alone of the result of disputes, but appeals are possible unlike in CentralizedArbitrator
+ */
+contract AppealableArbitrator is CentralizedArbitrator {
+    /** @dev Cost of appeal, as of now just multiplying this by two to create increasing cost for appeals
+     *  @param _disputeID ID of the dispute to be appealed. Not used by this contract.
+     *  @param _extraData Not used by this contract.
+     *  @return fee Amount to be paid.
+     */
+    function appealCost(uint _disputeID, bytes _extraData) public constant returns(uint fee) {
+        return arbitrationPrice*2;
+    }
+
+    /**
+     * @dev - remove owner as arbitrator since appeal was successful, this function assumes that the arbitrable is reassigned by Arbitrable
+     * @param _disputeID ID of the dispute to remove
+     */
+    function appealSuccessful(uint _disputeID) public {
+        Dispute dispute = disputes[_disputeID];
+        require(dispute.status==DisputeStatus.Appealable);
+        delete disputes[_disputeID];
+    }
+
+    /** @dev Give a ruling. UNTRUSTED.
+     *  @param _disputeID ID of the dispute to rule.
+     *  @param _ruling Ruling given by the arbitrator. Note that 0 means "Not able/wanting to make a decision".
+     */
+    function giveRuling(uint _disputeID, uint _ruling) public onlyOwner {
```

Comment:
The function calls directly the arbitrable contract, so the result is directly enforced and there is no time for appeal.